app/controllers/users_controller.rb

```diff
@@ -113,6 +113,17 @@ def extlogout
     render :extlogout, :layout => 'login'
   end
 
+  def test_mail
+    begin
+      user = find_resource(:edit_users)
+      MailNotification[:tester].deliver(:user => user, :email => params[:user_email])
```

Comment:
`params[:user_email]` can be blank for new users, I think you need to check if it's present or not so the UI doesn't report that sending the email to a blank address is successful.